include/utils.h

```diff
@@ -112,5 +118,20 @@ randomCategoriesLimits(int nbCategories, unsigned long int seed = time(NULL)) {
   sort(catLimits.begin(), catLimits.end());
   return catLimits;
 }
+/**
+ * Overloading << operator for std::unordered_map
+ *
+ * @param out ostream
+ * @param std::unordered_map  object
+ *
+ */
+template <typename K, typename V>
+std::ostream &operator<<(std::ostream &out, std::unordered_map<K, V> const &m) {
+  for (auto const &pair : m) {
+    out << "{, (" << pair.first << ": " << pair.second << "),";
```

Comment:
Isn't the { display as many times as the for loop is running whereas the closing } is displayed only once?